docs/integrations/labext.md

```diff
@@ -47,13 +47,19 @@ This makes use of [jovian.commit's](../jvn/commit) parameters to enable the user
 
 **Note:** By default the parameters are derived from [jovian.commit](https://jovian-py.readthedocs.io/en/latest/jvn/commit.html), changes to any parameter persists after commit.
 
+**Step 3:** Click on `Commit` to commit the notebook with following options.
+
 <img src="https://i.imgur.com/XdVkMPZ.png" class="screenshot" alt="jupyter extension commit options prompt">
 
-**Step 3:** Click on `Commit` to commit the notebook with following options.
+## Install Jovian Juputer Lab Extension
 
-## Enable or Disable the extension
+You can use NPM to install the extension by running the following command.
 
-By default, the Jovian Jupyter Lab Extension is enabled to the environment where jovian is installed.
+```
+$ Coming soon
+```
+
```

Comment:
This should be at the top.

Reply:
You can let mayank know to push this to the top of the page along with details, ill be merging this PR as of now.